specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -0,0 +1,3056 @@
+            "required": false,
+            "type": "string",
+            "collectionFormat": "csv",
+            "enum": [
+              "jobData"
+            ],
+            "x-ms-enum": {
+              "name": "ExpandQueryParamsType",
+              "modelAsString": true,
+              "values": [
+                {
+                  "name": "JobData",
+                  "value": "jobData",
+                  "description": "Use this value to expand the job with jobData property."
+                }
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/RadiologyInsightsJob"
+            },
+            "headers": {
+              "Retry-After": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "An opaque, globally-unique, server-generated string identifier for the request."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the status and details of the Radiology Insights job.": {
+            "$ref": "./examples/SuccessfulRadiologyInsightsResponse.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "RadiologyInsights_CreateJob",
+        "tags": [
+          "RadiologyInsights"
+        ],
+        "summary": "Create Radiology Insights job",
+        "description": "Creates a Radiology Insights job with the given request body.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique ID of the job.",
+            "required": true,
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 36,
+            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-_]+[a-zA-Z0-9]$"
+          },
+          {
+            "name": "expand",
+            "in": "query",
+            "description": "A comma separated list of related properties to be included in line with the job.",
+            "required": false,
+            "type": "string",
+            "collectionFormat": "csv",
```

Comment:
`collectionFormat` is not meaningful for `type: string`. You should either remove this or change to `type: array`.